### DISCUSSION TITLE
Cleanup for objects in ILEDITOR temp library

### DISCUSSION THREAD

**Discussion by cwbuck** [codefori/vscode-ibmi#2285] [IBM i] [answered]
Could someone help me understand what types of objects are getting created in the ILEDITOR? Our organization runs usage and size audits on our system to provide a report for objects that have not been used for a while or objects that are large in size. Often times, there are tons of PFs and DDMFs that appear on the report in the ILEDITOR library. Seems like there's even a lot of duplication with them.  Just wondering if anyone has a habit of cleaning them up and could provide insight as to what's appropriate to delete or if they all need to be left alone for people's extension to work properly. 

Thanks!

**Comment by worksofliam** [comment, marked as answer]
@cwbuck We use it to store:

* SQL aliases to source members
* Pls which are basically temporary listings
* Programs (for things like CL prompting) which are shared between users

If I were a system admin, I would regularly `DLTOBJ ILEDITOR/*ALL *PF`.

The files that are created are not important, and the programs and the aliases are more important. If you do delete them, they will be created again, but it means the clients will slow down as they will create again.

**Reply by cwbuck** [reply]
How about the DDMF objects?

**Reply by worksofliam** [reply]
@cwbuck those would be the aliases. Preferably keep them, but if it's an issue you can delete them - but they will be recreated as users open members.

**Reply by worksofliam** [reply]
@cwbuck those would be the aliases. Preferably keep them, but if it's an issue you can delete them - but they will be recreated as users open members.
